features/user_can_create_invoice.feature

```diff
@@ -0,0 +1,13 @@
+Feature: User can create invoice
+
+  Background:
+    Given the following user exist
+      | email         | password |
+      | user@user.com | password |
+
+    And I am logged in as "user@user.com"
+    And I am on the invoices page
+
+  Scenario: User can create an invoice
```

Comment:
But are you actually creating a user, can't see a create action in the controller